src/ui/public/notify/notify.less

```diff
@@ -77,6 +77,14 @@
     }
   }
 
+  .toast-banner {
+    // no flex
```

Comment:
I'll remove it, as long as it's acceptable to assign a className to an element that doesn't have any rules in the stylesheet. I just needed a way to differentiate this type of toaster from the others which have a lot of rules that come with the `.toast` class - I didn't want to override each rule.